lua/nvim-lsp-installer/server.lua

```diff
@@ -70,6 +71,24 @@ function M.Server:setup(opts)
     end
 end
 
+---Sets up the language server and attaches all open buffers.
+---@param opts table @The lspconfig server configuration.
+function M.Server:setup(opts)
+    self:setup_lsp(opts)
+    self:attach_buffers()
+end
+
+---Attaches this server to all current open buffers with a 'filetype' that matches the server's configured filetypes.
+function M.Server:attach_buffers()
+    log.debug("Attaching server to buffers", self.name)
+    local lsp_server = require("lspconfig")[self.name]
+    for _, bufnr in ipairs(vim.api.nvim_list_bufs()) do
+        log.fmt_trace("Attaching server=%s to bufnr=%s", self.name, bufnr)
```

Comment:
```suggestion
        log.fmt_debug("Attaching server=%s to bufnr=%s", self.name, bufnr)
```

Reply:
I think that might add too much noise. I think it's fine to assume vim.api.nvim_list_bufs() works as it should